.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  parser: "@typescript-eslint/parser", // Specifies the ESLint parser
+  parserOptions: {
+    ecmaVersion: 2020, // Allows for the parsing of modern ECMAScript features
+    sourceType: "module", // Allows for the use of imports
+    ecmaFeatures: {
+      jsx: true // Allows for the parsing of JSX
```

Comment:
I think we shouldn't have jsx support in this project too, it's not necessary, what do you think?

Reply:
Yeah true, Sorry I accidentally added react support too